app/services/plant_service.rb

```diff
@@ -17,9 +17,7 @@ def self.get_multiple_info(search_params)
   end
 
   def self.get_all_plants
-    response = conn.get('allplants')
-    JSON.parse(response.body, symbolize_names: true)
+    @@get_all_plants ||= JSON.parse(conn.get('allplants').body, symbolize_names: true) 
```

Comment:
the response from the microservice is stored in a class variable. using memoization, the call to the microservice only needs to be done once